crypto/s2n_ecc.c

```diff
@@ -28,12 +28,26 @@
 
 #define TLS_EC_CURVE_TYPE_NAMED 3
 
+struct s2n_ecc_named_curve s2n_ecc_curve_secp256r1 = {
```

Comment:
maybe making this a struct a `const` in the declaration itself is better, it will prevent the modification of the values downstream.

Reply:
fixed